### ISSUE TITLE
Questions about the precision in validation

### ISSUE THREAD

**Issue by Easquel** [CuriousAI/mean-teacher#35]
I trained the ResNet architecture (cifar_shakeshake26 in Pytorch version) on cifar-10 dataset with 1000 unlabeled images and 44000 labeled images (the resting 5000 images are used for validation) for about 180 epochs, setting the bach-size 256, unlabeled batch-size 62. 
But I observed that the validation precision (top 1) would first rise from 43% up to 50% and then fall to only 13% (began to fall after about 10 epochs) along the training process. I was so puzzled about this phenomenon. Besides, the precision in training always rise and never fall, why the validation precision would fall??

**Comment by LokLu** [comment]
@Easquel Have you solved it?
I come across with the same issue. My validation accuracy stays around 60% to the end of training while the training accuracy can go up to 100%.

**Comment by LokLu** [comment]
@tarvaina May you help the pytorch on cifar10?
I ran the code with pytorch0.3.1. The validation accuracy went up to 50% and then went down to 10~20%.

**Comment by Wangzheaos** [comment]
> 训练了ResNet体

should be 1000 labeled images and 44000 labeled images
